ydb/public/lib/ydb_cli/common/plan2svg.cpp

```diff
@@ -405,6 +405,31 @@ void TPlan::ResolveCteRefs() {
     }
 }
 
+void LoadJoinUnput(TStringBuilder& builder, const TString Name, const NJson::TJsonValue& node) {
```

Comment:
LoadJoinInput ?

Reply:
Clarify your question please

Reply:
> Clarify your question please

Did you mean `LoadJoinInput` Instead of `LoadJoinUnput`?
Looks like a typo